src/store/store.ts

```diff
@@ -1175,6 +1215,9 @@ export const useStore = defineStore('store', {
     setWindowWidth (width: number) {
       this.stateModel.windowWidth = width
     },
+    setAmalgamationType (type: AmalgamationTypes) {
+      this.stateModel.amalgamation.type = type
+    },
```

Comment:
For regular amalgamations, this will be pre-set in the draft filing that we start with here in Create UI.

For short-form amalgamations, the user will set this somehow (UI designs tbd).